photon-diagnostics/src/main/scala/com/linkedin/photon/ml/Evaluation.scala

```diff
@@ -64,40 +65,42 @@ object Evaluation extends Logging {
 
     // Compute regression facet metrics
     model match {
-      case r: Regression =>
+      case _: Regression =>
         val regressionMetrics = new RegressionMetrics(scoreAndLabel)
-        metrics ++= Map[String, Double](MEAN_ABSOLUTE_ERROR -> regressionMetrics.meanAbsoluteError,
-                                        MEAN_SQUARE_ERROR -> regressionMetrics.meanSquaredError,
-                                        ROOT_MEAN_SQUARE_ERROR -> regressionMetrics.rootMeanSquaredError)
```

Comment:
I think this is okay, but just want to point out that there's really nothing wrong with the original way. The warning in this case is a compiler bug.